modules/web/js/ballerina/diagram2/views/default/components/nodes/package-declaration-node.jsx

```diff
@@ -191,15 +137,19 @@ class PackageDeclarationNode extends React.Component {
         if (!value) {
             return;
         }
-        // value = 'import ' + value + ';\n';
         if (this.props.model.parent.isExistingPackage(value)) {
             const errorString = 'Package "' + value + '" is already imported.';
             log.debug(errorString);
             return;
         }
-        // Create import node
-        const importNode = NodeFactory.createImport({ package: value });
-        this.props.model.parent.addTopLevelNodes(importNode);
+        value = 'import ' + value + ';\n';
+        parseContent(value)
+            .then((jsonTree) => {
+                if (jsonTree.topLevelNodes[0]) {
+                    this.props.model.parent.addTopLevelNodes(TreeBuilder.build(jsonTree.topLevelNodes[0]));
```

Comment:
Same as above make sure the order is proper.

Reply:
Fixed with commit 637385f